index.js

```diff
@@ -14,6 +14,7 @@
  */
 function find(array, callback) {
   for (let element of array) {
+   return undefined;
```

Comment:
having a return in your for loop is going to cause your code to not iterate the array and instead immediately return `undefined`